### PR TITLE
Reordering the website nav for user guides

### PR DESCRIPTION
**Pull request by hallaroo**

### CODE REVIEW

**Comment by deepthi** [comment]
I like this idea. However, I see two issues:
* Some of the basic user guides we point people to (`MoveTables` and `Resharding`) are not easily visible. `Resharding` in particular doesn't even show up on the User guide page, it is hidden under `Advanced Configuration` -> `More`
* There are links elsewhere in the website that are broken by this change. E.g. `Getting Started` -> `Local Install` links to `MoveTables` at the end. That is right now a 404. The other `Getting Started` guides also point to `MoveTables`, they might also be broken. You will need to test every link from every page in the preview to ensure that all links work.

File: content/en/docs/reference/programs/vtexplain.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ aliases: ['/docs/reference/vtexplain/']
 
 `vtexplain` is a command line tool which provides information on how Vitess plans to execute a particular query. It can be used to validate queries for compatibility with Vitess.
 
-For a user guide that describes how to use the `vtexplain` tool to explain how Vitess executes a particular SQL statement, see [Explaining how Vitess executes a SQL statement](https://vitess.io/docs/user-guides/vtexplain/).
+For a user guide that describes how to use the `vtexplain` tool to explain how Vitess executes a particular SQL statement, see [Explaining how Vitess executes a SQL statement](https://vitess.io/docs/user-guides/sql/vtexplain/).
```
Comment (hallaroo) [inline content/en/docs/reference/programs/vtexplain.md]: Not working doublecheck.

File: content/en/docs/user-guides/configuration-advanced/authorization.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ behavior of ACLs.  Let's review these:
    users.
  * `-queryserver-config-enable-table-acl-dry-run`:  Set to `true` to check the
    table ACL at runtime, and only emit the
-   [TableACLPseudoDenied](../configuring-components/#tableaclallowed-tableacldenied-tableaclpseudodenied)
+   [TableACLPseudoDenied](../configuration-basic/configuring-components/#tableaclallowed-tableacldenied-tableaclpseudodenied)
```
Comment (hallaroo) [inline content/en/docs/user-guides/configuration-advanced/authorization.md]: That section doesn't exist anymore

Comment (hallaroo) [reply content/en/docs/user-guides/configuration-advanced/authorization.md]: Substituting with monitoring as that contains the only reference to TableACLPseudoDenied

**Comment by hallaroo** [comment]
I'm redoing this elsewhere as this got old and was missing user guides

**Pull request closed without merging**